src/morse/api.clj

```diff
@@ -101,3 +101,11 @@
   ([token chat-id options audio]
    (assert-file-type audio ["mp3"]) 
    (send-file token chat-id options audio "/sendAudio" "audio" "audio.mp3")))
+
+
+(defn send-sticker
+  "Sends a sticker to the chat"
+  ([token chat-id sticker] (send-sticker token chat-id {} sticker))
+  ([token chat-id options sticker]
+   (assert-file-type image ["webp"])
```

Comment:
Oops! This is what you get when you rename variables at the last minute and do not re-test it. I've now fixed it.